dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -59,25 +56,27 @@
  */
 final class QueuedChannel implements LimitedChannel {
     private static final Logger log = LoggerFactory.getLogger(QueuedChannel.class);
-    private static final Executor DIRECT = MoreExecutors.directExecutor();
 
     private final Deque<DeferredCall> queuedCalls;
     private final LimitedChannel delegate;
+    private final String serviceName;
     // Tracks requests that are current executing in delegate and are not tracked in queuedCalls
     private final AtomicInteger numRunningRequests = new AtomicInteger(0);
```

Comment:
hmm, I don't think this is actually used, our ActiveRequestInstrumentationChannel provides the same data.

Reply:
I think `numRunningRequests` can be deleted now. I don't think it's ever read.